src/lsptoolshost/razor/razorEndpoints.ts

```diff
@@ -65,6 +82,59 @@ export function registerRazorEndpoints(
         }
     );
 
+    registerRequestHandler<FoldingRangeParams, FoldingRange[]>(FoldingRangeRequest.method, async (params) => {
```

Comment:
Not a huge deal but likely going to have some conflicts with https://github.com/dotnet/vscode-csharp/pull/8225 because I moved things into `cohosting` and `noncosting`

Reply:
Conflicts I can deal with, I'm mostly impressed you managed to get a non-flaky test run :P

Reply:
been trying to track this down, and I think its an issue with the .net install tool - testing a change to reduce flakiness here - https://github.com/dotnet/vscode-csharp/pull/8233

Reply:
in the meantime, if you have a PR that is essentially ready to merge, ping me and I can help merge if its only hitting the known flakiness.

Reply:
Thanks. I seem to have mostly had issues with one of the linux non-omnisharp tests, and touch wood they've just passed on this PR so fingers crossed.